examples/copper/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "copper"
```

Comment:
If you publish this crate it might be colliding with an existing one

Reply:
not publishing the examples